packages/ui/src/components/Title/Title.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import s from './Title.module.css';
+import { useActiveQueue } from '../../hooks/useActiveQueue';
 
-interface TitleProps {
-  name?: string;
-  description?: string;
-}
+export const Title = () => {
+  const queue = useActiveQueue();
```

Comment:
as hooks can be composed from other hooks, this hook doesnt need to have any arguments, it will resolve automatically which is the active queue